tunnels.py

```diff
@@ -23,7 +23,10 @@
 @click.option('--region')
 @click.option('-U', '--user')
 def cli(stack_name, port, jump_host, region, user):
-    out = subprocess.check_output(['senza', 'instances', '--output=json', stack_name])
+    if region:
+        out = subprocess.check_output(['senza', 'instances', '--output=json', stack_name,'--region='+region])
```

Comment:
could you keep it a bit more DRY? i.e. construct the Senza command line before and append the --region option when necessary